packages/redis-v5/commands/upgrade.js

```diff
@@ -24,8 +24,8 @@ module.exports = {
 
     await cli.confirmApp(context.app, context.flags.confirm, `WARNING: Irreversible action.\nRedis database will be upgraded to ${cli.color.configVar(version)}. This cannot be undone.`)
     await cli.action(`Requesting upgrade of ${cli.color.addon(addon.name)} to ${version}`, async function () {
-      await api.request(`/redis/v0/databases/${addon.name}/upgrade`, 'POST', { version: version })
-      cli.action.done(`upgrade has started!`)
+      let res = await api.request(`/redis/v0/databases/${addon.name}/upgrade`, 'POST', { version: version })
+      cli.action.done(res.message)
```

Comment:
What are the various messages that can get returned here? I'm not finding anything in DevCenter.
I'm concerned that we might confuse users.

Reply:
For now this is what we are thinking: https://github.com/heroku/shogun/blob/54d5c5dd409bd41a0086028bd621814a5e8d22bc/lib/shogun/endpoints/client/redis.rb#L98. I can put this into the tests if that is helpful?